File: benchmarks/bench-expo/ios/BenchExpoModule.swift
import ExpoModulesCore

/** NitroBenchmarks' MyExpoModule. */
public class BenchExpoModule: Module {
  public func definition() -> ModuleDefinition {
    Name("BenchExpo")

    Function("addNumbers") { (a: Double, b: Double) in
      return a + b
    }
    Function("addStrings") { (a: String, b: String) in
      return a + b
    }
  }
}

File: benchmarks/bench-expo/android/src/main/java/dev/lucent/bench/expo/BenchExpoModule.kt
package dev.lucent.bench.expo

import expo.modules.kotlin.modules.Module
import expo.modules.kotlin.modules.ModuleDefinition

/** NitroBenchmarks' MyExpoModule. */
class BenchExpoModule : Module() {
  override fun definition() = ModuleDefinition {
    Name("BenchExpo")

    Function("addNumbers") { a: Double, b: Double -> a + b }
    Function("addStrings") { a: String, b: String -> a + b }
  }
}
